src/node/types.h

```diff
@@ -39,6 +41,13 @@ struct BlockCreateOptions {
      * larger CompactSize encoded lengths.
      */
     size_t block_reserved_weight{8000};
+
+    /*
+     * This is a sane minimum value for block_reserved_weight.
+     * Reserving weight for coinbase transaction and block header
+     * below this value will be prevented.
+     */
+    const uint32_t minimum_block_reserved_weight{2000};
```

Comment:
131eb4e630f958f1b5330bb9d170716c33655880: this should be a constant, not part of `BlockCreateOptions`.

E.g. `MINIMUM_BLOCK_RESERVED_WEIGHT{2000}` defined next to `DEFAULT_BLOCK_MAX_WEIGHT`.

Reply:
Done